### ISSUE TITLE
Add menu to change mode

### ISSUE THREAD

**Issue by nbrachet** [nbrachet/FlyDrones.iOS#13]
When the user clicks on the current mode, a "bubble menu" should appear with the option to change the mode -- ie. a list of the following modes: STABILIZE, ALT_HOLD, LOITER, RTL, LAND, DRIFT, POSHOLD.
The current mode shouldn't be in the menu.

**Comment by nbrachet** [comment]
Note: mode is changed by sending a SET_MODE message:

```
GCS: SEQ:182 SYSID:252 COMPID:  1 - SET_MODE: custom_mode=LOITER target_system=1 base_mode=CUSTOM_MODE
```

**Comment by nab0y4enko** [comment]
Done in 0.1.4.3

**Comment by nbrachet** [comment]
menu should close once a heartbeat message is received

**Comment by nab0y4enko** [comment]
Fixed in version 0.1.5
I recorded a video with simulation of the process
https://youtu.be/A95kKF0Ak4Y

**Comment by nbrachet** [comment]
Nope. Menus still stay displayed.
